ecs/spotfleet/spotrequest-builder/build-spotrequest.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+if [ "$#" -ne 3 ]; then
+    echo 'Must be called with input and output file and number of subnets e.g  build-spotrequest.sh small.csv spotrequest.tf 3'
```

Comment:
This effectively hardcodes the required number of subnets. So you might as well remove the subnet_count from variables.

Reply:
That bit of bash scripting checks for 3 parameters - you could use it to generate a spotfleet for any number of subnets..  e.g. build-spotrequest.sh small.csv spotrequest.tf 2

But yes the current single configuration file that is used is hardcoded for 3 subnets

Reply:
Ok, so we'd have 3 different configuration files for 1, 2 and 3 subnets?

Reply:
I am not sure if we need to spend the time and add the complexity for the full range of configurations that would be nice...  If we did then we could have 9 pre-made request files.
Small , Medium and Large for each of 1, 2 or 3 subnets.

I was hoping we could see how it went with just the 1 we have (small on 3 subnets) and hope that terraform has improved when we need to extend this

Reply:
That sounds like a good idea :) But in the mean time, we can just remove the `subnet_count` at any rate, since it has a hardcoded requirement of 3x.